### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by amashutikov**
[DEMO LINK](https://amashutikov.github.io/react_decompose/)

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -1 +1,42 @@
...
+  opacity: 0.7;
+  padding: 0 0.5em;
+  color: black;
+  transition: opacity 0.2s ease-in-out;
+}
+
+.navigation__link:hover,
+.navigation__link:focus {
+  opacity: 1;
+}
```
Comment (33Orange) [inline src/components/Header/Header.css:34]: You may also create a folder for the Navigation component and move navigation there.

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
-// Create a Header function returning the HTML of header block
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```
Comment (33Orange) [inline src/components/Header/Header.jsx:12]: Create separate component for navigation.
Check task description: 
`You may also create a folder for the Navigation component and move navigation there.`

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done.

Let's create separate component for navigation and code will be ready

**Review by silchyn — CHANGES_REQUESTED** [review]
Fix these `GitHub Actions` errors

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+import React from 'react';
+import './Header.css';
+import Nav from '../Navigation/Navigation';
 
-// Create a Header function returning the HTML of header block
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <Nav />
```
Comment (33Orange) [inline src/components/Header/Header.jsx:9]:
```suggestion
      <Navigation />
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done.

But where the navigation file ?

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**